src/components/organisms/Badges/Badges.tsx

```diff
@@ -12,22 +11,47 @@ import {
 import { getUserRef } from "api/profile";
 
 import { UserVisit } from "types/Firestore";
-import { ContainerClassName } from "types/utility";
 import { AnyVenue, isVenueWithRooms } from "types/venues";
 
 import { WithId } from "utils/id";
 import { isTruthy, notEmpty } from "utils/types";
 
+import { useWorldEdit } from "hooks/useWorldEdit";
+
 import { BadgeImage } from "./BadgeImage";
 
 import "./Badges.scss";
 
-export const Badges: React.FC<
-  {
-    user: WithId<User>;
-    currentVenue: WithId<AnyVenue>;
-  } & ContainerClassName
-> = ({ user, currentVenue, containerClassName }) => {
+const findVenueAndRoomByName = (
```

Comment:
We can export/import this as a helper function from another file and improve the reusability. It looks like something that might be needed in another file as well.

Reply:
If it is needed somewhere else, we can do it. I think I've already done more alterations that might look like OOS, so just kept it all in that file for no. I do think it will get refactored if/when accounting gets changed from venue(s) to world or similar